plugins/logistics.cpp

```diff
@@ -391,7 +431,7 @@ static const struct BadFlags {
     uint32_t get_bad_flags() {
         df::item_flags flags;
         #define F(x) flags.bits.x = true;
-        F(forbid); F(garbage_collect); F(hostile); F(on_fire);
+        F(garbage_collect); F(hostile); F(on_fire);
```

Comment:
bah, I didn't see that F(artifact) is already in BadFlags. You don't need to check for it in the processors. sorry!

Reply:
Missed that as well... thanks again!